cmd/systrack/main.go

```diff
@@ -3,49 +3,48 @@ package main
 import (
 	"log"
 
+	"go.mozilla.org/systrack"
```

Comment:
We use `go.mozilla.org` for all internal packages and app. I set it up in https://github.com/mozilla-services/gomoz/commit/a6b23a73638416045415f80acb4073ed4c1c6690